submission/proxy_test.go

```diff
@@ -121,3 +122,48 @@ func TestProxyRefreshRootsErr(t *testing.T) {
 		}
 	}
 }
+
+func TestCTPolicyTypeFromString(t *testing.T) {
+	testCases := []struct {
+		name      string
+		from      string
+		wantPlc   CTPolicyType
+		errRegexp *regexp.Regexp
+	}{
+		{
```

Comment:
nit: Looks like it can fit 1 line. Same for the next test.
`		{name: "chrome", from: "chrome", want: ChromeCTPolicy},`